metricbeat/module/docker/event/_meta/fields.yml

```diff
@@ -0,0 +1,52 @@
+- name: event
+  type: group
+  description: >
+    Docker event
+  release: ga
+  fields:
+    - name: status
+      type: keyword
+      description: >
+        Event status
+    - name: id
+      type: alias
+      path: event.id
+      migration: true
+      description: >
+        Event id when available
+    - name: from
+      type: keyword
+      description: >
+        Event source
+    - name: type
+      type: alias
+      path: event.kind
+      migration: true
+      description: >
+        The type of object emitting the event
+    - name: action
```

Comment:
You set migration: true here. Is this an alias we only want to be enabled if migration is enabled or have it always? Same for the other aliases.

Reply:
I want it to stay, I'm guessing I should not have put migration there :innocent: ? I can open a PR to remove them :+1:

Reply:
yes, otherwise they will disappear.

Reply:
@exekias BTW this is a more general discussion if we want to have fields in ECS only or also the "original" name around.